base/deprecated.jl

```diff
@@ -803,6 +803,32 @@ function transpose(x)
     return x
 end
 
+# Deprecate full in favor of convert. Please see #17066, #17079, and #17082.
```

Comment:
Good catch! Moved deprecations below the 0.6 line. Also preemptively rewrote some `@deprecate`'ions with explicit `depwarns`. Thanks!